### PR TITLE
Add newest version of Font Awesome to Bootstrap 3 pages

### PR DESCRIPTION
**Pull request by biyeun**
## Technical Summary
This adds the newest version of Font Awesome to Bootstrap 3 pages. This also removes the old version from `package.json`.

### Safety story
Deploying to staging and testing by going to various pages and seeing how the icons look. From what I've seen locally, the newest version is very much backwards-compatible with earlier font-awesome versions and old style formats are respected.

### Automated test coverage
No

### QA Plan
No

### Rollback instructions

leaving this unchecked due to bootstrap diffs causing issues if reverted
- [ ] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/apps/hqwebapp/templates/hqwebapp/includes/bootstrap3/core_stylesheets.html [hunk 1]
```diff
@@ -6,10 +6,10 @@
           rel="stylesheet"
           media="all"
           href="{% static 'hqwebapp/less/bootstrap.less' %}" />
-    <link type="text/less"
+    <link type="text/css"
           rel="stylesheet"
           media="all"
-          href="{% static 'font-awesome/less/font-awesome.less' %}" />
+          href="{% static '@fortawesome/fontawesome-free/css/all.min.css' %}" />
```
Comment (millerdev) [inline corehq/apps/hqwebapp/templates/hqwebapp/includes/bootstrap3/core_stylesheets.html:12]: Is `@fortawesome` correct, or should that be `@fontawesome`?

**Review by millerdev — APPROVED** [review]
Looks good except for a potential typo. Obviously that needs to be fixed before merging if it's a typo.

File: corehq/apps/hqwebapp/templates/hqwebapp/includes/bootstrap3/core_stylesheets.html (hunk 1 above)
Comment (biyeun) [reply corehq/apps/hqwebapp/templates/hqwebapp/includes/bootstrap3/core_stylesheets.html:12]: this is correct. New massive js projects now have these `@` libraries that are the "main library" with all the plugin libraries beneath it. Font Awesome's is correctly FoRt Awesome lol

**Review by millerdev — APPROVED** [review]
Given the volume of changes and that they were most likely all automated, I did not review the most recent changes thoroughly.

**Review by millerdev — APPROVED** [review]

**Review by millerdev — APPROVED** [review]

**Pull request merged**